File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.java
class Solution {
    public int[] topKFrequent(int[] nums, int k) {
        Map<Integer,Integer> mp = new HashMap<>();
        for(int num:nums){
            mp.put(num,mp.getOrDefault(num,0)+1);
        }
        PriorityQueue<int[]> pq = new PriorityQueue<>((a,b)->Integer.compare(b[1],a[1]));
        for(Map.Entry<Integer,Integer> entry:mp.entrySet()){
            pq.add(new int[]{entry.getKey(),entry.getValue()});
        }
        int[] ans = new int[k];
        while(k > 0 && !pq.isEmpty()){
            int [] fr = pq.poll();
            ans[k-1] = fr[0];
            k--;
        }
        return ans;
    }
}

File: Data Structures & Algorithms/missing-number/submission-1.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int st = 0;
        int ans = 0;
        for(int i = 0;i<nums.size();i++){
            ans ^= (st^nums[i]);
            st++;
        }
        ans ^= st;
        return ans;
    }
};
